testr/packages.py

```diff
@@ -187,7 +187,10 @@ def run_tests(package, tests):
             interpreter = test['interpreter']
 
             logger.info('Running {} {} script'.format(interpreter, test['file']))
-            logfile = Tee(test['file'] + '.log')
+            if test['file'].endswith('.sh'):
```

Comment:
Can we just do `logfile = Tee(Path(test['file']).with_suffix('.log')))`?  (Assuming `Tee` accepts a Path). The asymmetry of treating `.sh` doesn't spark joy.

Reply:
yes, I agree it is ugly. I will try your proposal.

Reply:
Tee passes the path directly to `open` and nothing else, so this line would work.

The log_file is used in another place, which uses other functions, so I have to decide how to change it. Pathlib is not used in this module. It uses os.path heavily.